language/move-model/src/model.rs

```diff
@@ -2046,6 +2100,32 @@ impl<'env> ModuleEnv<'env> {
             .disassemble()
             .expect("Failed to disassemble a verified module")
     }
+
+    /// Return true if the module has any global, module, function, or struct specs
+    pub fn has_specs(&self) -> bool {
```

Comment:
Similar here--I *think* I've covered all the kinds of specs one can add to a module, but could definitely be wrong.

Reply:
There is a field f.get_spec().on_impl on function specs which contains specs embedded in code.